web/src/lib/components/ui/tags-multiple-select/tags-multiple-select.svelte

```diff
@@ -2,17 +2,14 @@
   import * as Select from '$lib/elements/ui/select';
   import type { Tag } from '$lib/sdk/fetch-client';
   import type { Selected } from 'bits-ui';
-  import { onMount } from 'svelte';
 
   export let tags: Tag[];
-  let selectedTags: Selected<string>[] = [];
-  export let selectedTagIds: unknown[];
-  export let entityTags: Tag[];
+  export let selectedTagIds: string[];
 
-  onMount(() => {
-    selectedTags = entityTags.map((tag) => ({ label: tag.name, value: tag.id }));
-    selectedTagIds = selectedTags.map((tag) => tag.value);
-  });
+  let selectedTags: Selected<string>[] = tags
+    .filter((tag) => selectedTagIds.includes(tag.id))
+    .map((tag) => ({ label: tag.name, value: tag.id }));
+  selectedTagIds = selectedTags.map((tag) => tag.value);
```

Comment:
You can make this reactive, so replace the `let selectedTagIds` declaration with `$: selectedTagIds = selectedTags.map((tag) => tag.value)` and then you can also get rid of the `onSelectedChange` on the select below.